src/trace-agent.js

```diff
@@ -35,6 +35,7 @@ var traceAgent;
 function TraceAgent(config, logger) {
   this.config_ = config;
   this.logger = logger;
+  this.plugins = config ? config.plugins : {};
```

Comment:
Should this line be:
```
this.plugins = config && config.plugins ? config.plugins : {};
```
?  Otherwise, its possible for `this.plugins` to still be `undefined` which will cause `src/hooks.js#activate` to fail.